pinot-common/src/main/java/org/apache/pinot/common/utils/DatabaseUtils.java

```diff
@@ -155,4 +157,14 @@ public static String extractDatabaseFromQueryRequest(
     String database = databaseFromHeaders != null ? databaseFromHeaders : databaseFromOptions;
     return Objects.requireNonNullElse(database, CommonConstants.DEFAULT_DATABASE);
   }
+
+  public static String extractDatabaseFromTableName(String tableName) {
```

Comment:
```suggestion
  public static String extractDatabaseFromFullyQualifiedTableName(String tableName) {
```
nit: to make it more clear. Also could you please add a small Javadoc here?